plugins/field-bitmap/src/field-bitmap.ts

```diff
@@ -636,4 +641,7 @@ Blockly.Css.register(`
 .controlButton {
   margin: 5px 0;
 }
+.blocklyDropDownContent{
+  max-height: unset;
```

Comment:
Unfortunately adding this css will break unrelated dropdowns, e.g. you can get this with the test toolbox:
![Screenshot 2024-05-16 at 5 01 32 PM](https://github.com/google/blockly-samples/assets/8573958/cd5fd8ed-faa8-4826-aa9a-c7576a01cb85)
where the dropdown is so long it extends past the workspace.

I think you probably want to unset it just for dropdowns that contain the bitmap field, not for all dropdowns just because this plugin is loaded. However, that is easier said than done. There's a pseudo-selector you can use for this but it's pretty new / still a working draft https://caniuse.com/css-has

While you can add an additional class to the dropdownDiv (or add an inline style) when you show the editor, there's no corresponding method that gets called when the field's editor is hidden, so there's no opportunity to remove it. So I can't actually think of a solution to this right now :/

I would remove this from this PR and you mentioned some other ideas you had for fixing the height you could follow up with. You could also just warn users in the readme not to set the height greater than the max-height of dropdowns (which by default is 300px)

Reply:
Yikes, good catch!  It looks like we can potentially handle this by adding the additional class during `dropdownCreate` and removing it during `dropdownDispose`. See newest commit. This seems to work as expected when switching between two different dropdown field types' editors. Thoughts?